manifests/params.pp

```diff
@@ -75,7 +76,9 @@
     'Debian': {
       # system packages
       $system_packages = ['libicu-dev', 'python2.7','python-docutils',
-                          'libxml2-dev', 'libxslt1-dev','python-dev']
+                          'libxml2-dev', 'libxslt1-dev','python-dev', 'cmake', 'pkg-config', 'libkrb5-dev', 'ruby-execjs']
+                          
+      $redis_packages = ['redis-server']
```

Comment:
@blkperl I suggest, I had not thought of before. Is there an hint for choose of an dependency?

The following redis module looks like the most used module: https://forge.puppetlabs.com/thomasvandoren/redis